code/datums/mob_hud.dm

```diff
@@ -682,9 +682,12 @@ GLOBAL_LIST_INIT_TYPED(huds, /datum/mob_hud, list(
 	holder.icon_state = "hudblank"
 	holder.overlays.Cut()
 
-	if(mob_flags & MUTINEER)
-		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudmutineer")
-		return
+	if(mob_flags & MUTINY_MUTINEER)
+		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudmutineer", pixel_y = 12)
+	else if(mob_flags & MUTINY_LOYALIST)
+		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudloyalist", pixel_y = 12)
+	else if(mob_flags & MUTINY_NONCOMBAT)
+		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudnoncombat", pixel_y = 9)
```

Comment:
Should probably add an early return to each of these as prior. e.g.
```suggestion
	if(mob_flags & MUTINY_MUTINEER)
		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudmutineer", pixel_y = 12)
		return
	else if(mob_flags & MUTINY_LOYALIST)
		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudloyalist", pixel_y = 12)
		return
	else if(mob_flags & MUTINY_NONCOMBAT)
		holder.overlays += image('icons/mob/hud/marine_hud.dmi', src, "hudnoncombat", pixel_y = 9)
		return
```
otherwise when it attempts to add the mutiny overlay it'll also redo their SEA !NEW! icon & their squad icons and stuff-- and I think it might cause duplicates of them.

Reply:
Intentional so people can still see who's what role. Previous to this change the mutiny symbols overwrote the squad hud.
It won't duplicate their huds as the first part of this proc clears their HUD icon to rebuild it from scratch.